boards/arm/stm32f030_demo/stm32f030_demo_defconfig

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Zephyr Kernel Configuration
+CONFIG_ARM=y
+CONFIG_SOC_SERIES_STM32F0X=y
+
+# Platform Configuration
+CONFIG_SOC_STM32F030X4=y
+CONFIG_BOARD_STM32F030_DEMO=y
+
+# General Kernel Options
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
+
+# Kernel Options due to Low Memory (4k)
+CONFIG_MAIN_STACK_SIZE=640
+CONFIG_IDLE_STACK_SIZE=200
+CONFIG_ISR_STACK_SIZE=512
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Pinmux Driver
+CONFIG_PINMUX=y
+
+# GPIO Controller
+CONFIG_GPIO=y
+CONFIG_GPIO_STM32_PORTD=n
```

Comment:
Since UART and Led and are allusing GPIO A, why not deactivating GPIO ports B & C as well?
This does not prevent enabling them for other use cases, but this would give some room to run extra samples in current zephyr base.